src/rdkafka_aux.h

```diff
@@ -128,4 +128,46 @@ void rd_kafka_Node_destroy(rd_kafka_Node_t *node);
 
 void rd_kafka_Node_free(void *node);
 
+/**
+ * @brief Represents a topic partition result with topic,
+ *        partition, error code, and error string.
+ *
```

Comment:
Remove field name. Check older documentation and follow the same format wherever possible.